### PR TITLE
Honda: brazilian HR-V 2023 fingerprint

### PR DESCRIPTION
**Pull request by AlexandreSato**
***** Template: Fingerprint *****

With the new dynamic alt brake detection introduced by #31500 the port of #30235 that created a new separate class without alt brake is no longer needed.

There is a small difference in wheelbase and mass that I don't know if is significant:
|      | American | Global |
| --- | --- | --- |
| mass | 1446kg | 1442kg |
| wheelbase | 2665mm | 2610mm |
| height | 1611mm | 1590mm |

**Car**
Honda, HR-V 2023 from @kotarou [Ktrtkd](https://github.com/Ktrtkd)

**Route**
```
147613502316e718|2024-02-26--14-49-16
```
```
147613502316e718|2024-02-26--14-31-49
```

### CODE REVIEW

**Comment by sshane** [comment]
Is there PR this replaces? Can you give a short run-down of what's different with this global car?

**Comment by sshane** [comment]
Are there any alpha or stock longitudinal issues? i.e. not braking for leads at low speeds or did they resolve themselves?

File: selfdrive/car/honda/fingerprints.py [hunk 1]
```diff
@@ -1307,24 +1307,34 @@
...
     ],
     (Ecu.srs, 0x18da53f1, None): [
       b'77959-3V0-A820\x00\x00',
+      b'77959-3M0-K840\x00\x00',
     ],
     (Ecu.combinationMeter, 0x18da60f1, None): [
       b'78108-3V1-A220\x00\x00',
+      b'78109-3M0-M310\x00\x00',
+    ],
+    (Ecu.fwdRadar, 0x18DAB0F1, None): [
```
Comment (sshane) [inline selfdrive/car/honda/fingerprints.py:1324]: huh, why was there no fwdRadar for this car?

Comment (sshane) [reply selfdrive/car/honda/fingerprints.py:1324]: I don't see this FW for this user, only this which looks like an alt address for the radar (1 digit off on the part number prefix)

image

Comment (AlexandreSato) [reply selfdrive/car/honda/fingerprints.py]: removed and tested!

**Comment by AlexandreSato** [comment]
> Is there a PR this replaces? Can you give a short run-down of what's different with this global car?

done!

**Comment by sshane** [comment]
make sure to give the title a descriptive name, like Honda: global HR-V model_year fingerprint

**Comment by AlexandreSato** [comment]
> make sure to give the title a descriptive name, like Honda: global/brz/thai HR-V model_year fingerprint. Not sure which this one is

Thanks for the lineup! Fixed, this is the Brazilian version.

**Comment by sshane** [comment]
@Ktrtkd can you try master? I want to get updated fingerprint results to make sure it can fingerprint without the comma power.

**Comment by Ktrtkd** [comment]
> @Ktrtkd can you try master? I want to get updated fingerprint results to make sure it can fingerprint without the comma power.

Sure, So basically is to test master and do not use comma power? If so I will do and post here later.

**Comment by sshane** [comment]
One without, and one with is good.

**Comment by Ktrtkd** [comment]
> One without, and one with is good.

Well here it is
With comma power 147613502316e718/00000001--ce406cf8a7
Without 147613502316e718/00000000--442ee72203

**Comment by sshane** [comment]
thanks! you can run without a comma power if you'd like, it will still fingerprint

**Pull request merged**